### DISCUSSION TITLE
Better shirt quality

### DISCUSSION THREAD

**Discussion by Letterus** [elementary/store#177] [Product Ideas]
Bought some t-shirts, but they were not 100% cotton and I don't really like to wear them, because they are thin, smell and look cheap (shipped within the EU). Some nice, thick 100% cotton shirts would be great.

**Comment by cassidyjames** [comment]
@Letterus thanks for the feedback! We chose tri-blend because they're soft and have gotten feedback that people prefer the light and soft feel. Regarding smell, we've heard from the printers (and I believe they include a card in the package stating) that the print process can leave a sort of vinegar smell but that does dissipate and then go away completely upon washing.

However, I've also heard some people prefer heavy cotton, so we could consider that which I think would address your comments here. :)
